src/main/flight/gps_rescue.h

```diff
@@ -21,6 +21,10 @@
 
 #include "pg/pg.h"
 
+#define GPS_RESCUE_THROTTLE_P_SCALE 0.000125f
```

Comment:
If these are not used anywhere outside of `gps_rescue.c` then they should be defined in there.

Reply:
Ok